engine/src/main/java/org/terasology/world/chunks/internal/ChunkImpl.java

```diff
@@ -113,8 +114,9 @@ public Vector3i getPosition() {
     }
 
     @Override
-    public org.joml.Vector3i getPosition(org.joml.Vector3i dest) {
-        return dest.set(chunkPos.x,chunkPos.y,chunkPos.z);
```

Comment:
wondering if it makes sense to use the method as is like this and then optimize it in post. Vector3ic is preferred this dest is not necessary. the constant type could just be wrapped into another vector as follows below. dest does not add anything else since there is nothing else on the back-end beyond just returning the position. Might make sense to keep it like this and drop it at the end when this whole class is migrated over and deprecate the staging methods here. 

```
new Vector3i(block.getPosition());
new Vector3i().add(block.getPosition());
```